app/mailers/participant_mailer.rb

```diff
@@ -3,21 +3,20 @@
 class ParticipantMailer < ApplicationMailer
   PARTICIPANT_REMOVED_BY_STI = "ab8fb8b1-9f44-4d27-8e80-01d5d70d22f6"
 
-  PARTICIPANT_ADDED_TEMPLATES = {
-    ect: "50ee41e5-06b9-41cf-9afd-d5bc4db356c4",
-    mentor: "e0198213-c09d-41aa-8197-b167e495e49d",
-    sit_mentor: "7e7d3fdb-41f5-4e04-a4ae-acf92e8fefe6",
+  PARTICIPANT_TEMPLATES = {
+    ect_cip: "i3bfad27e-e7af-4b53-94cb-af36362f43d8",
+    ect_fip: "3d3dbe1a-5836-4781-836f-e546beae253e",
+    mentor_cip: "3721ec20-ba5a-4424-8114-0e009b5bdbda",
+    mentor_fip: "ba3d4caf-5ef8-4a14-9e79-b7719780da09",
+    sit_mentor_cip: "7aa80a9c-e486-42e8-92c8-7f970459d37d",
+    sit_mentor_fip: "0b7f850f-f26a-4e62-9fc5-17fbd8286e49",
   }.freeze
 
-  DETAIL_REMINDER_TEMPLATES = {
-    ect: "0bf633c3-54c9-4150-b1fb-57748376aed1",
-    mentor: "691eb49b-f23f-49dd-b799-f8efdd5e010f",
-    sit_mentor: "fcf85c00-6d96-48a6-bf27-ea09c61f0eee",
-  }.freeze
+  WHAT_EACH_PERSON_DOES_URL = "https://manage-training-for-early-career-teachers.education.gov.uk/pages/what-each-person-does"
```

Comment:
` Rails.application.routes.url_helpers.page_url(page: :"what-each-person-does", host: Rails.application.config.domain)` please, we don't hardcode the service URL. We may also want to add some UTM params to this?